### DISCUSSION TITLE
Create a second ceph cluster use Rook

### DISCUSSION THREAD

**Discussion by zhang871622105** [rook/rook#13916] [Q&A]
I want to create two clusters use Rook,   https://rook.io/docs/rook/latest/Storage-Configuration/Advanced/ceph-configuration/   Deploying a second cluster，
second cluster:  I create a new namespace and  modify namespace of common.yaml ，operator.yaml ， but the second cluster do not start success

**Comment by travisn** [comment]
@zhang871622105 Try this topic to use alternate namespaces. https://rook.io/docs/rook/latest/Storage-Configuration/Advanced/ceph-configuration/#using-alternate-namespaces
